RubeesEat/Model/DB/DbTransactionExtensions.cs

```diff
@@ -0,0 +1,21 @@
+using System.Data;
+using System.Data.Common;
+
+namespace RubeesEat.Model.DB;
+
+public static class DbTransactionExtensions
+{
+    public static DbCommand? CreateCommand(this DbTransaction transaction, string sql)
+    {
+        //Rider says transaction.Connection could be null
+        //(because we assign the connection to the transaction outside of this function)
+        //should we check for null and return null in that case?
+        var command = transaction.Connection?.CreateCommand();
+        if (command == null) return null;
```

Comment:
Please no nullable return of `DbCommand`. This would imply that returning `null` is supported and we would need to check this fact on all usage sites. We can do a check and fail here for the connection to improve the error if the connection is not set, or we just assume that it is set and use `transaction.Connection!.CreateCommand()` instead